modules/auxiliary/admin/http/sysaid_file_download.rb

```diff
@@ -0,0 +1,150 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Auxiliary
+
+  include Msf::Auxiliary::Report
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "SysAid Help Desk Arbitrary File Download",
+      'Description' => %q{
+        This module exploits two vulnerabilities in SysAid Help Desk that allows
+        an unauthenticated user to download arbitrary files from the system. First an
+        information disclosure vulnerability (CVE-2015-2997) is used to obtain the file
+        system path, and then we abuse a directory traversal (CVE-2015-2996) to download
+        the file. Note that there are some limitations on Windows: 1) the information
+        disclosure vulnerability doesn't work; 2) we can only traverse the current drive,
+        so if you enter C:\afile.txt and the server is running on D:\ the file will not
+        be downloaded. This module has been tested with SysAid 14.4 on Windows and Linux.
+        },
+      'Author' =>
+        [
+          'Pedro Ribeiro <pedrib[at]gmail.com>' # Vulnerability discovery and MSF module
+        ],
+      'License' => MSF_LICENSE,
+      'References' =>
+        [
+          [ 'CVE', '2015-2996' ],
+          [ 'CVE', '2015-2997' ],
+          [ 'OSVDB', 'TODO' ],
+          [ 'OSVDB', 'TODO' ],
+          [ 'URL', 'https://raw.githubusercontent.com/pedrib/PoC/master/generic/sysaid-14.4-multiple-vulns.txt' ],
+          [ 'URL', 'TODO_FULLDISC_URL' ]
+        ],
+      'DisclosureDate' => 'Jun 3 2015'))
+
+    register_options(
+      [
+        OptPort.new('RPORT', [true, 'The target port', 8080]),
+        OptString.new('TARGETURI', [ true,  "SysAid path", '/sysaid']),
+        OptString.new('FILEPATH', [false, 'Path of the file to download (escape Windows paths with 4 back slashes)', '/etc/passwd']),
```

Comment:
Ah this, yeah a double-slash should be one slash. For example, here I have this OptString datastore option:

```
msf auxiliary(test) > show options

Module options (auxiliary/admin/http/test):

   Name         Current Setting  Required  Description
   ----         ---------------  --------  -----------
   Some_option                   no        Test
```

Notice what I set it with a double-slash, it's converted to a single slash:

```
msf auxiliary(test) > set Some_option C:\\test.txt
Some_option => C:\test.txt
```

And when I print_status it:

```
msf auxiliary(test) > run

[*] C:\test.txt
```

So double-slash should be it.